akd/benches/directory.rs

```diff
@@ -79,7 +79,7 @@ fn history_generation<TC: NamedConfiguration>(c: &mut Criterion) {
 
                 // generate for the most recent 10 updates
                 let label = AkdLabel::from("User 1");
-                let params = akd::HistoryParams::MostRecentInsecure(5);
+                let params = akd::HistoryParams::MostRecent(5);
```

Comment:
I think we should actually keep the name with the "Insecure" tag. Adding deprecation tags doesn't necessarily imply that the functionality is insecure, as this could be confusing to callers.